pkg/tools/sandbox/node.go

```diff
@@ -116,13 +119,22 @@ func (n *Node) NewForwarder(
 		Name: nse.Name,
 		URL:  serveURL,
 	}
+	nseClient := chain.NewNetworkServiceEndpointRegistryClient(
+		registryclient.NewNetworkServiceEndpointRegistryClient(ctx, CloneURL(n.NSMgr.URL),
+			registryclient.WithNSEAdditionalFunctionality(recvfd.NewNetworkServiceEndpointRegistryClient()),
+			registryclient.WithDialOptions(dialOptions...),
+		),
+	)
+	nsClient := registryclient.NewNetworkServiceRegistryClient(ctx, CloneURL(n.NSMgr.URL), registryclient.WithDialOptions(dialOptions...))
 	entry.restartableServer = newRestartableServer(ctx, n.t, entry.URL, func(ctx context.Context) {
 		entry.Endpoint = endpoint.NewServer(ctx, generatorFunc,
 			endpoint.WithName(entry.Name),
 			endpoint.WithAdditionalFunctionality(
 				append(
-					additionalFunctionality,
-					clienturl.NewServer(CloneURL(n.NSMgr.URL)),
+					append([]networkservice.NetworkServiceServer{
+						discover.NewServer(nsClient, nseClient),
+						roundrobin.NewServer(),
```

Comment:
@edwarnicke Should we add chains/forwarder with these two chain elements?

Reply:
> Should we add chains/forwarder with these two chain elements?

Could you say more... I don't quite follow...

Reply:
we can add a new pkg `forwarder` into `sdk/pkg/networkservice/chains/`.

So currently, it seems to me we have motivation because many forwarders may be interested in default SDK chain elements such as `roundrobin`, `disocver`, `recvfd`, `sendfd`. So we can add a default preset for them.

Reply:
Hmm... lets try getting the existing forwarders going first

Reply:
Because discover/roundrobin are kind of bespoke to our p2p forwarders... p2mp forwarders may chose to do something quite different.

Reply:
@edwarnicke Load balancing sound like option :)